test/python/tools/visualization/test_visualization.py

```diff
@@ -13,7 +13,8 @@
 from inspect import signature
 
 from qiskit import QuantumRegister, ClassicalRegister, QuantumCircuit
-from qiskit.tools.visualization import _utils, circuit_drawer
+from qiskit.visualization import _utils
+from qiskit.tools.visualization import circuit_drawer
```

Comment:
```suggestion
from qiskit.visualization import circuit_drawer
```

Reply:
I left it like this to test the backwards compat worked, but I we probably want to move it too.